crates/fluvio-hub-protocol/src/package_meta.rs

```diff
@@ -195,6 +196,23 @@ impl PackageMeta {
             None
         }
     }
+
+    pub fn published_at(&self) -> Result<DateTime<Utc>> {
```

Comment:
Instead of adding this to `fluvio-hub-protocol` as an impl fn of PackageMeta, can it be added to `fluvio-hub-util` to the `PackageMetaExt`  This keeps the deps in the protocol crate lower and less complex.

Reply:
Makes sense!